zdm/iteration.py

```diff
@@ -835,7 +851,8 @@ def missing_cube_likelihoods(grids,surveys,todo,outfile,norm=True,psnr=True,star
     f.close()
 
 
-def cube_likelihoods(grids,surveys,psetmins,psetmaxes,npoints,
+def cube_likelihoods(grids,surveys, #psetmins,psetmaxes,npoints,
```

Comment:
I'm going to state right now that even in my implementation I needed to run this to check it was behaving properly - viewing the code didn't help me much, so while I'm ticking this one off, it will really need its own test.